setup.py

```diff
@@ -28,6 +28,9 @@ def get_version():
         'scikit-learn>=0.19.1',
         'lalsuite>=6.70',
         'gwsurrogate',
+        'qnm',
```

Comment:
The qnm package is also only needed for an example notebook and thus testing

Reply:
Done